openjdk/src/test/java/org/conscrypt/AbstractSessionContextTest.java

```diff
@@ -96,6 +96,28 @@ public void testImplicitRemovalOfOldest() {
         assertSessionContextContents(toArray(c, d), toArray(a, b));
     }
 
+    @Test
+    public void testRemoveIfSingleUse() {
```

Comment:
Might also want a test about putting in two single-use tickets, and making sure you get 2 back out.

Reply:
Done, though it was added to ClientSessionContextTest, since that's the version that stores multiples for the same host/port pair.